Sources/Basics/FileSystem/AbsolutePath.swift

```diff
@@ -38,13 +42,17 @@ public typealias TSCAbsolutePath = TSCBasic.AbsolutePath
 /// that evaluates the tilde; the `cd` command receives an absolute path).
 public struct AbsolutePath: Hashable, Sendable {
     /// Root directory (whose string representation is just a path separator).
-    public static let root = Self(TSCAbsolutePath.root)
+    public static let root = Self(FilePath("/"))
```

Comment:
It looks like there's something called `FilePath.Root` which can also represent the multiple roots on Windows. Seems like we should switch to that, at least eventually.

Reply:
yes